src/main/java/com/deavensoft/paketomat/center/CenterServiceImpl.java

```diff
@@ -38,4 +41,16 @@ public void deletePackageById(Long id) {
     public void deleteAll() {
         centerRepository.deleteAll();
     }
+
+    @Override
+    public void updateStatus(Long code, Status status) {
+        List<Package> packages = centerRepository.findAll();
+        for (Package p : packages) {
+            if(p.getCode().equals(code)){
+                p.setStatus(status);
+                centerRepository.save(p);
+                //System.out.println("Paket sa kodom: " + p.getCode() + " je updateovan: " + p.getStatus());
```

Comment:
Programmers should not comment out code as it bloats programs and reduces readability.

- [ ] Remove it